### PR TITLE
manager view with the analytical rep(chart) is created 

### PR DESCRIPTION
**Pull request by sadrisya**
click on the team from the side bar will show the manager dash board

### CODE REVIEW

File: app/views/shared/components/_fixed_side_bar.html.erb [hunk 1]
```diff
@@ -24,7 +24,7 @@
           <% end %>
         </li>
         <li class="side-bar-item">
-          <%= link_to "#", class: "side-bar-item-inner" do %>
+          <%= link_to manager_path, class: "side-bar-item-inner" do %>
```
Comment (deepakkumarnd) [inline app/views/shared/components/_fixed_side_bar.html.erb:27]: can you skip this change for now ? I will update this separately. We need to make this change for mobile sidebar too.

File: config/routes.rb [hunk 2]
```diff
@@ -81,6 +81,8 @@
   # for now dashboard path and root path are same.
   get "error_401" => "pages#unauthorized"
   get "dashboard" => "dashboards#index"
+  get "manager" => "dashboards#index_manager"
```
Comment (deepakkumarnd) [inline config/routes.rb:84]: skip this route, I will add the necessary changes to make this work automatically

File: app/views/dashboards/_index_manager_dashboard_desktop.html.erb [hunk 3]
```diff
@@ -0,0 +1,72 @@
+<%= desktop_view do %>
```
[the remark this answers is not in the document]
Comment (sadrisya) [reply app/views/dashboards/_index_manager_dashboard_desktop.html.erb]: <%= desktop_view do %> is replaced with <%= render "shared/components/desktop_layout" do %>

File: app/views/shared/components/_fixed_side_bar.html.erb (hunk 1 above)
Comment (sadrisya) [reply app/views/shared/components/_fixed_side_bar.html.erb]: reverted

File: config/routes.rb (hunk 2 above)
Comment (sadrisya) [reply config/routes.rb]: removed the path

**Review by deepakkumarnd — APPROVED** [review]

**Pull request merged**